.travis.yml

```diff
@@ -46,6 +46,8 @@ matrix:
         - brew link bdw-gc
         - brew install jq
         - brew install re2
+        - brew install llvm@3.7
+      env: PATH="$PATH:/usr/local/opt/llvm@3.7/bin"
```

Comment:
To keep the macOS build on the same version, however, we could also change it so CI builds for oldest and newest supported LLVM version.